front/src/modules/ui/themes/colors.ts

```diff
@@ -94,6 +94,7 @@ export const color: { [key: string]: string } = {
   red40: '#bf5a5a',
   red30: '#f09797',
   red20: '#fbc5c5',
+  'red20-1': '#fed8d8',
```

Comment:
let's not introduce a new color here. The palette is off and we plan to update it. Please use red20 for now! We will fix the palette this week